src/KrigingEstimators.jl

```diff
@@ -7,7 +7,7 @@ module KrigingEstimators
 using GeoStatsBase
 using Variography
 
-using LinearAlgebra: Factorization, lu, cholesky, issuccess, ⋅
+using LinearAlgebra: Factorization, Symmetric, lu, cholesky, issuccess, ⋅
```

Comment:
Could you please revert this change? I understood we will be including these optimizations in a separate PR right?